src/rod/sdf/common.py

```diff
@@ -89,7 +89,9 @@ def transform(self) -> npt.NDArray:
         )
 
     @staticmethod
-    def from_transform(transform: npt.NDArray, relative_to: str = None) -> "Pose":
+    def from_transform(
+        transform: npt.NDArray, relative_to: str | None = None
+    ) -> "Pose":
```

Comment:
Here you can also import annotations and use update the returned type

Reply:
Done in https://github.com/ami-iit/rod/pull/39/commits/137415fdccb4680ee4cbe0bf971e97338ae87fa1